srt/server.go

```diff
@@ -277,7 +274,8 @@ func (s *ServerImpl) registerForStats(ctx context.Context, conn *srtConn) {
 func (s *ServerImpl) GetStatistics() []*relay.StreamStatistics {
 	streams := s.relay.GetStatistics()
 	for _, stream := range streams {
-		stream.URL = fmt.Sprintf("srt://%s?streamid=play/%s", s.config.Address, stream.Name)
+		//TODO how to decide which Address to use
```

Comment:
I think using the first address should be fine as the user can reorder the addresses accordingly.